source/containers.tex

```diff
@@ -11236,20 +11260,29 @@
 \indexlibraryglobal{as_bytes}%
 \begin{itemdecl}
 template<class ElementType, size_t Extent>
-  span<const byte, Extent == dynamic_extent ? dynamic_extent : sizeof(ElementType) * Extent>
+  span<const byte, @\seebelow@>
```

Comment:
These changes are missing in the synopsis.

Reply:
Not really, since the synopsis just spells out `Extent == dynamic_extent ? dynamic_extent : sizeof(ElementType) * Extent` in full, which is exactly what the _see below_ expands to.

I pushed a new commit that does it differently anyway.